src/runtime/type.ts

```diff
@@ -1195,19 +1195,22 @@ interface TypeVariableInfoLiteral {
   name: string;
   canWriteSuperset?: TypeLiteral;
   canReadSubset?: TypeLiteral;
+  resolveToMaxType: boolean;
 }
 
 export class TypeVariableInfo {
   name: string;
   _canWriteSuperset?: Type|null;
   _canReadSubset?: Type|null;
   _resolution?: Type|null;
+  _resolveToMaxType: boolean;
```

Comment:
if this is intended to be private, mark it as private. Either way, don't use the underscore (this was an old pattern that we are slowly getting rid of).